assets/src/edit-story/components/form/dateTime/dateTime.js

```diff
@@ -52,6 +55,21 @@ function DateTime({
     hours: is12Hour ? initialHours % 12 || 12 : initialHours,
     date: selectedTime,
   });
+
+  const previousFocus = useRef(document.activeElement);
+  const handleClose = useCallback(
```

Comment:
We still need focus trapping - you can shift-tab out of the overlay and move focus to some random place in the editor (the overlay will close). Shift-tab should just tab to the last focusable element inside the component and vice-versa. We have this focus trapping in our dialog component, so you can probably steal the functionality from there.

Reply:
@barklund Using the Modal to replace all Popups didn't seem that straightforward currently, added a potential focus trapping util instead, that should fix the calendar accessibility as well, thoughts? https://github.com/google/web-stories-wp/pull/4677/commits/bd80e493d8e7cc00c7ae9020e913cdcdcea53e34